src/plugins/discover/public/application/main/services/discover_saved_search_container.test.ts

```diff
@@ -200,7 +168,7 @@ describe('DiscoverSavedSearchContainer', () => {
     });
 
     it('Error thrown on persistence layer bubbling up, no changes to the initial saved search ', async () => {
-      mockSaveSavedSearch.mockImplementation(() => {
+      discoverServiceMock.savedSearch.get = jest.fn().mockImplementation(() => {
```

Comment:
this file is looking so much better now 👍  `mockSaveSavedSearch` what a name, it's hiding something very well, to make it green we need to 
```suggestion
      discoverServiceMock.savedSearch.save= jest.fn().mockImplementation(() => {
```